frontend/src/scenes/experiments/ExperimentView/StartDatePicker.tsx

```diff
@@ -0,0 +1,48 @@
+import { LemonButton } from '@posthog/lemon-ui'
+import { DatePicker } from 'lib/components/DatePicker'
+import { dayjs } from 'lib/dayjs'
+import { useState } from 'react'
+
+interface StartDatePickerProps {
+    currentStartDate: string
+    updateStartDate: (_: string) => void
+}
+
+export function StartDatePicker({ currentStartDate, updateStartDate }: StartDatePickerProps): JSX.Element {
+    const [isSelectorOpen, setIsSelectorOpen] = useState(false)
+
+    return (
+        <div>
+            {isSelectorOpen ? (
+                <DatePicker
+                    onSelect={(date: dayjs.Dayjs) => {
+                        updateStartDate(date.toISOString())
+                    }}
+                    showTime={false}
```

Comment:
```suggestion
                    showTime={true}
                    showSeconds={false}
```

I think we should support minute-level precision when setting the start date. This could be useful when you accidentally send some testing events that should not be included in the results. Shifting the date by a few minutes will remove them.

Reply:
Done!